### DISCUSSION TITLE
Open the list edit modal box, click the cancel button, and execute unmountTableAction twice

### DISCUSSION THREAD

**Discussion by killjin** [filamentphp/filament#7443] [Help]
![image](https://github.com/filamentphp/filament/assets/2804854/35a7cf0d-3ac2-4b58-a122-74445d706d53)

![image](https://github.com/filamentphp/filament/assets/2804854/da2a9787-b4e1-4ef7-a758-cb30b4c24836)

**Comment by pboivin** [comment]
Is this causing any issues for you?

**Comment by killjin** [comment]
![image](https://github.com/filamentphp/filament/assets/2804854/b8719d44-43fa-45fe-9250-f631e6fcb0d3)
![image](https://github.com/filamentphp/filament/assets/2804854/702a7e3f-fb37-4ddd-9c40-9e2be7bd6569)

Should be the second time a console error is generated.

**Reply by pboivin** [reply]
@killjin Sure looks like a bug, can you open an issue?
